system/Database/BaseConnection.php

```diff
@@ -854,6 +854,14 @@ public function table($tableName)
         return new $className($tableName, $this);
     }
 
+    /**
+     * Returns a new instance of the BaseBuilder class without a table.
+     */
+    public function newQuery(): BaseBuilder
+    {
+        return $this->table('crunch')->from([], true);
```

Comment:
What's with the `crunch` table name? This seems to be getting a build instance using `crunch` as the table name, not "a new instance of BaseBuilder without a table" as the docblock reads. 

I'm assuming this is just to fill a fake table name need? If that's the case let's use some type of random string here instead to ensure we don't accidentally conflict with a table someone might have.

Reply:
All attempts to change the BaseBuilder code to create a really clean instance are just marked as "breaking changes".
I hope that someday we will remove the table name from the constructor. 
But so far, as I wrote above, we have no other way to create an instance other than passing a temporary fake table name to the constructor.
In fact, the fake table name will not conflict with the one already existing in the application, but if you think so, what can you say about using the ``uniqid()`` function instead of a fixed table name.